src/model/stationDAO.js

```diff
@@ -0,0 +1,35 @@
+import LineDAO from "./lineDAO.js";
+
+export default class StationDAO {
+  constructor() {
+    this.localStorage = window.localStorage;
+  }
+  getAllStations() {
+    if (!this.localStorage.getItem("stations")) return [];
+    return this.localStorage.getItem("stations").split(",");
+  }
+  addStation(stationName) {
+    let stations = this.getAllStations();
+    if (stations.indexOf(stationName) != -1) return null;
+    stations.push(stationName);
+    this.localStorage.setItem("stations", stations.toString());
+    return stationName;
+  }
+  isStationInLine(stationName) {
+    let lineDAO = new LineDAO();
+    let lines = lineDAO.getAllLines();
+    let lineArray = Object.keys(lines);
+    let ret = false;
+    lineArray.forEach(line => {
+      if (lines[line].includes(stationName)) ret = true;
+    });
+    return ret;
+  }
```

Comment:
```suggestion
  isStationInLine(stationName) {
    let lineDAO = new LineDAO();
    let lines = lineDAO.getAllLines();
    let lineArray = Object.keys(lines);
    return lineArray.filter(line => lines[line].includes(stationName)).length > 0;
  }
```

filter를 사용하면 조금 더 줄일 수 있을것같아 리뷰했습니다.

Reply:
훨씬 깔끔하네요 감사합니다!